lib/src/widgets/video_track_renderer.dart

```diff
@@ -52,10 +53,18 @@ class VideoTrackRenderer extends StatefulWidget {
 
 class _VideoTrackRendererState extends State<VideoTrackRenderer> {
   rtc.RTCVideoRenderer? _renderer;
+  bool _rendererReady = false;
   EventsListener<TrackEvent>? _listener;
   // Used to compute visibility information
   late GlobalKey _internalKey;
 
+  Future<rtc.RTCVideoRenderer?> _initializeRenderer() async {
```

Comment:
seems like it's not possible for _renderer to be null? would be better to reflect that in the method signature.